src/Microsoft.DotNet.Interactive.Http.Tests/HttpKernelTests.cs

```diff
@@ -1194,7 +1194,7 @@ public async Task invalid_option_produces_error()
             """;
 
         var result = await kernel.SendAsync(new SubmitCode(code));
-        result.Events.Should().ContainSingle<DiagnosticsProduced>().Which.Diagnostics.Should().ContainSingle().Which.Message.Should().Be("The supplied option 't' in the expression '$datetime 'yyyy-MM-dd' -1 t' is not supported.");
+        result.Events.Should().ContainSingle<DiagnosticsProduced>().Which.Diagnostics.Should().ContainSingle().Which.Message.Should().Be("The supplied option 't' in the expression '$datetime 'yyyy-MM-dd' -1 t' is not supported. The following options are supported: ms, s, m, h, d, w, M, Q, y. See https://aka.ms/http-date-time-format for more details.");
```

Comment:
do we need this to be localised?

Reply:
Yes, all of the parser diagnostics need to be localized. And since the parser package is source-only, we need to do be able to localize in both the current repo and the internal repo. We have discussed this before and have some ideas around how to do this. There will be a follow up PR around this before long.